app/pods/publications/publication/controller.js

```diff
@@ -89,8 +79,8 @@ export default class PublicationController extends Controller {
     return this.statusOptions.find((statusOption) => statusOption.id === this.model.publicationFlow.get('status.id'));
   }
 
-  get getPublicationType() {
-    return this.typeOptions.find((typeOption) => typeOption.id === this.model.publicationFlow.get('type.id'));
+  get getPublicationMode() {
```

Comment:
Gezien de vroegere code is het begrijpelijk dat je als nieuwe naam `getPublicationMode` gekozen hebt, maar weet dat we dat liever niet doen. Met `get publicationMode()  { ...` kan je na de definitie de eigenschap gewoon gebruiken als `this.publicationMode`. Dat is duidelijker en bevat geen dubbele informatie. Je mag het hier zo laten. We passen het aan in een andere pull-request.